lib/src/cli.rs

```diff
@@ -512,6 +515,27 @@ async fn usroverlay() -> Result<()> {
         .into());
 }
 
+/// Implementation of `bootc build commit`
+/// async fn lint() -> Result<()> {
+fn lint() -> Result<()> {
+    // TODO: Check if 1 or more folder under the /lib/modules/ directory
+    println!("Checking if we are in the container");
+    if ostree_ext::container_utils::is_ostree_container()? {
+        let modules_path = cap_std::fs::Dir::open_ambient_dir("lib/modules", cap_std::ambient_authority())?;
+        for entry in modules_path.entries()? {
+            let dir = entry?;
+            let fileName= dir.file_name();
+            debug(fileName);
+
+        }
+        return Ok(());
+    }
+    anyhow::bail!(
```

Comment:
This is OK as is, but I tend to prefer doing "early return" style to avoid rightward drift.  So like this:

```
if !is_ostree_container()? {
   anyhow::bail!(...)
}

...expected case here...
}

Reply:
100%, we have done this as well. left it at the bottom as we were still not solid on the logic that was needed for verifying everything. Good catch :)